src/postgres/config.ts

```diff
@@ -0,0 +1,6 @@
+// Enter these values to enable postgres account view
+export const config = {
+    user: '<username>@<postgres_account_name>', //Add username@accountname here, username is the Azure username
```

Comment:
We prefer to keep master brach in a ship-ready state and this username/password logic seems pretty far from ship-ready. We have two options:
1. Wait for you to implement better username/password logic to merge this PR
1. Hide all of the Postgres stuff behind a feature flag, implement username/password stuff in a separate PR, and remove the feature flag once that's done

I guess I would prefer option 2 since this PR is already pretty big and I don't want to delay merging even longer. Here's an example PR where I'm _removing_ a feature flag: https://github.com/microsoft/vscode-azurestorage/pull/562. The important part is just the VS Code setting `azureStorage.enableViewInFileExplorer` which you should emulate